### PR TITLE
Externalize messages for external plugins.

### PR DESCRIPTION
**Pull request by rtibbles**
Set the locale_data_folder for plugins outside of the kolibri app to inside their own app.

### CODE REVIEW

File: kolibri/core/webpack/hooks.py [hunk 1]
```diff
@@ -152,13 +152,23 @@ def webpack_bundle_data(self):
...
                 "static_url_root": getattr(django_settings, 'STATIC_URL'),
-                "locale_data_folder": os.path.join(getattr(django_settings, 'LOCALE_PATHS')[0], 'en', 'LC_FRONTEND_MESSAGES'),
+                "locale_data_folder": self.locale_data_folder,
                 "version": self.version,
             }
         else:
             logger.warn("{src_file} not found for plugin {name}.".format(src_file=self.src_file, name=self.unique_slug))
 
     @property
+    def locale_data_folder(self):
```
Comment (benjaoming) [inline kolibri/core/webpack/hooks.py:162]: This is only pertaining frontend locale data?

**Comment by rtibbles** [comment]
This particular pr is, yes. Would be good for external plugins to add their
locale folders to django's on activation. Do we need to do anything special
to make that happen?

File: kolibri/core/webpack/hooks.py (hunk 1 above)
Comment (benjaoming) [reply kolibri/core/webpack/hooks.py:162]: I was wondering if the property name was too general, but in a sense - even though it is configurable - it's supposed to be used as a convention.

Comment (benjaoming) [reply kolibri/core/webpack/hooks.py:162]: ...and so the property name is a bit more internal and not so important

Comment (benjaoming) [reply kolibri/core/webpack/hooks.py:162]: The property is not supposed to be overwritten, and plugins aren't supposed or likely to reconfigure the conventional locale data path, isn't that so?

Comment (rtibbles) [reply kolibri/core/webpack/hooks.py:162]: Correct, and only external plugins can overwrite the property that this is derived from (we could remove that feature to enforce the convention that the `locale` folder always lives in the root of the external plugin folder).

**Comment by benjaoming** [comment]
@rtibbles application locale folders are already added automatically when applications are in `INSTALLED_APPS` - so plugins are, too :)

**Comment by rtibbles** [comment]
Cool beans!

**Pull request merged**